text/0000-icons.md

```diff
@@ -8,14 +8,15 @@
 
 ## Purpose
 
-Provide Pluralsight icons for easy use in JavaScript applications.
-Maintain consistent iconography across applications.
-Reduce maintenance efforts and improve efficiency across UI teams.
+- Provide Pluralsight icons for easy use in JavaScript applications.
+- Maintain consistent iconography across applications.
+- Reduce maintenance efforts and improve efficiency across UI teams.
 
 ## Tools
 
-- [SVGO](https://www.npmjs.com/package/svgo)
-- [svg-to-jsx](https://www.npmjs.com/package/svg-to-jsx)
+- [svgo](https://www.npmjs.com/package/svgo)
+- [Babel](https://babeljs.io/)
```

Comment:
We'll need to make sure the workspace uses it's own babel config that includes this and extends the root config.

Reply:
If we switch to typescript, we probably won't need Babel, but would probably need the same for its configuration.  Should it be noted here about the configuration extending the root?